provisioning/miniprov/hare_mp/main.py

```diff
@@ -181,7 +181,10 @@ def unsupported_feature(url: str):
         logging.error('Error reporting hare unsupported features (%s)', error)
 
 
-def _start_consul(utils: Utils, stop_event: Event, hare_local_dir: str):
+def _start_consul(utils: Utils,
+                  stop_event: Event,
+                  hare_local_dir: str,
+                  url: str):
     log_dir = f'{hare_local_dir}/consul/log'
```

Comment:
Let's create these directories in a separate function, may be `create_consul_env()` and invoke it in prepare stage before invoking _start_consul().

Reply:
Done